### PR TITLE
new console messages polish

### PR DESCRIPTION
**Pull request by souporserious**
Further refinement of messages to match previous console.

### CODE REVIEW

**Comment by jasonLaster** [comment]
Oh man, this looks so much better 👍 

One small visual glitch, but otherwise amazing
* filtered-complex-array-preview-linux.png
* filters-side-menu-open-linux.png

**Comment by souporserious** [comment]
@jasonLaster should be good now! Was trying to get it up before you looked, but git was taking forever to push screenshots 😛

File: packages/bvaughn-architecture-demo/components/console/CurrentTimeIndicator.module.css [hunk 1]
```diff
@@ -2,4 +2,6 @@
   width: 100%;
   height: 2px;
   background-color: var(--color-current);
+  position: absolute;
+  z-index: 5;
```
Comment (bvaughn) [inline packages/bvaughn-architecture-demo/components/console/CurrentTimeIndicator.module.css:6]: This caused a visual regression:
Screen Shot 2022-09-16 at 2 36 21 PM

Any idea why it was necessary? Looks like if I remove it, it fixes the regression (and doesn't seem to obviously break anything)

Comment (bvaughn) [reply packages/bvaughn-architecture-demo/components/console/CurrentTimeIndicator.module.css:6]: Removed in #7758

File: packages/bvaughn-architecture-demo/components/console/renderers/shared.module.css [hunk 2]
```diff
@@ -2,26 +2,35 @@
...
   padding-left: 1.5rem;
+  border-top: 1px solid var(--background-color-contrast-1);
   border-bottom: 1px solid var(--background-color-contrast-1);
   position: relative;
 }
 .ErrorRow {
   background-color: var(--background-color-error);
   color: var(--color-error);
+  border-color: var(--border-color-error);
+  z-index: 5;
```
Comment (bvaughn) [inline packages/bvaughn-architecture-demo/components/console/renderers/shared.module.css:16]: I think this also caused a regression:
Screen Shot 2022-09-16 at 2 56 12 PM

**Pull request merged**